app/presenters/withdrawable_presenter.rb

```diff
@@ -0,0 +1,37 @@
+class WithdrawablePresenter
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def withdrawal_notice_component
+    {
+      title: withdrawal_notice_title,
+      description_govspeak: content_item.withdrawal_notice["explanation"]&.html_safe,
+      time: withdrawal_notice_time,
+      lang: I18n.locale.to_s == "en" ? false : "en",
```

Comment:
We're not testing the "en" branch (I think? we're definitely not testing one of the branches here)